content/configuration/node.md

```diff
@@ -19,7 +26,9 @@ This is an object where each property is the name of a Node global or module and
 - `true`: Provide a polyfill.
 - `"mock"`: Provide a mock that implements the expected interface but has little or no functionality.
 - `"empty"`: Provide an empty object.
-- `false`: Provide nothing. Code that expects this object to be defined may crash.
+- `false`: Provide nothing. Code that expects this object may crash with a `ReferenceError`.
+  Code that attempts to import the module using `require('modulename')` may
+  trigger a `Cannot find module "modulename"` error.
```

Comment:
I added explicit error messages so that search engines can find the documentation more easily.